### PR TITLE
Add aFxs tvl

### PR DESCRIPTION
**Pull request by jianfengxiaojianfeng**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/concentrator: 
        
 
  
    
    
    Error: unable to verify the first certificate
    Truncated error stack:
    
        at TLSSocket.onConnectSecure (node:_tls_wrap:1535:34)
        at TLSSocket.emit (node:events:513:28)
        at TLSSocket._finishInit (node:_tls_wrap:949:8)
        at TLSWrap.ssl.onhandshakedone (node:_tls_wrap:730:12)

**Comment by llamatester** [comment]
The adapter at projects/concentrator exports TVL: 
        
 
 
    ethereum                  89.98 M
    
    total                    89.98 M

File: projects/concentrator/index.js [hunk 1]
```diff
@@ -29,7 +31,9 @@ const replacements = [
 
 async function tvl(timestamp, block) {
   let balances = {}
-
+  const initInfo = await fetchURL('http://concentrator-api.aladdin.club/data/aladdin/initInfo');
```
Comment (g1nt0ki) [inline projects/concentrator/index.js:34]: hi, can you compute this using on-chain calls instead?

Comment (jianfengxiaojianfeng) [reply projects/concentrator/index.js:34]: > hi, can you compute this using on-chain calls instead?

code is complex

Comment (g1nt0ki) [reply projects/concentrator/index.js:34]: @jianfengxiaojianfeng can you explan what is happening a bit, I can try to convert it on chain.

**Comment by llamatester** [comment]
The adapter at projects/concentrator exports TVL: 
        
 
 
    ethereum                  91.05 M
    
    total                    91.05 M

File: projects/concentrator/index.js (hunk 1 above)
Comment (jianfengxiaojianfeng) [reply projects/concentrator/index.js]: compute TVL on-chain

Comment (g1nt0ki) [reply projects/concentrator/index.js]: @jianfengxiaojianfeng thanks for turning it onchain

**Pull request merged**